website/docs/components/badge/partials/specifications/design-guidelines.md

```diff
@@ -1,46 +1,37 @@
-## When to use
-
-- To indicate status, such as "Running", "Applied", "Errored", etc.
-- As feature flags, such as "In Preview", "Beta", "New", etc.
-- For categorizations, such as Product Lines and Account Levels.
-- Keyboard shortcut hints, such as "Esc".
-- for collection enumeration or version number, use [BadgeCount](/components/badge-count/overview)
-
-## When not to use
-
-- For a list of metadata elements (like selected filters), consider using a [Tag](/components/tag/overview).
-- For dismissable elements, consider using a [Tag](/components/tag/overview).
-- For longer status messages, consider an [Alert](/components/alert/overview).
-- Don’t use a link within a Badge, consider moving the link outside of the Badge.
-- Don’t use BadgeCount for anything non-numeric, except version numbers (ie. “v1.2.0”).
-
----
-
 ## Anatomy
 
 ![Anatomy of the badge component](/assets/components/badge/badge-anatomy.png)
 
-#### Text
+| Element          | Usage                                                                |
+|------------------|----------------------------------------------------------------------|
+| Text             | Required                                                             |
+| Color            | Optional                                                             |
+| Size             | Optional                                                             |
+| Type             | Optional                                                             |
+| Icon             | Optional, but required on Status badges (Success, Warning, Critical) |
+| isIconOnly       | Optional                                                             |
+
+# TODO: Keep or delete below?
+
```

Comment:
I didn't know if it was ok to delete the below content. (Is the table essentially meant to replace it? I was following: https://github.com/hashicorp/design-system/blob/main/wiki/Website-Editing-existing-content.md#specifications)

Reply:
based on the Alert as a reference I understood this to mean that the content below would be deleted

Reply:
🤔  but the content here is more involved than what I saw when I did this for tag

Reply:
Thanks, yeah that's partly why I wasn't sure since the content seems fairly detailed.

Reply:
I think at least I'll comment it out for now...

Reply:
I think the designers would be able to give you a better answer than I have 😅

Reply:
No worries, thanks! If you are ok with the rest would you mind approving?

Reply:
I moved the content and renamed related md files.